app/utils/PasswordInput.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+interface PasswordInputProps {
+  value: string | null;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
this can be simplified to 
`onChange: (password: string) => void`
if that helps (as long as u change line 25 accordingly to 
`onChange={(e)=>onChange(e.target.value)}`